debian/varnish.init

```diff
@@ -88,7 +88,7 @@ reload_varnishd() {
     log_daemon_msg "Reloading $DESC" "$SERVICE"
     # Workaround for the reload script to make it work with instances
     export DAEMON_OPTS
-    if /usr/share/varnish/reload-vcl -q; then
+    if /usr/share/varnish/varnishreload -q; then
```

Comment:
`varnishreload` doesn't have a `-q` option.

Reply:
I fixed this, thanks.